src/04-modality-specific-files/10-eye-tracking.md

```diff
@@ -0,0 +1,187 @@
+
+### Sidecar JSON document (`*_eyetrack.json`)
+
+Generic fields MUST be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "TaskName": ("REQUIRED", "A RECOMMENDED convention is to name resting state task using labels beginning with `rest`."),
+   }
+) }}
+
+SHOULD be present: 
+
+For consistency between studies and institutions, we encourage users to extract the values of these fields from the actual raw data. Whenever possible, please avoid using ad hoc wording.
+
+{{ MACROS___make_metadata_table(
+   {
+      "InstitutionName": "RECOMMENDED",
+      "InstitutionAddress": "RECOMMENDED",
+      "Manufacturer": (
+         "RECOMMENDED",
+         "Manufacturer of the eye-tracking system (for example "
+         '`"SR-Research"`, `"Tobii"`, `"SMI"`, `"Gazepoint"`, '
+         '`"Pupil Labs"`, `"Custom built"`, `"Other"`) '
+      ),
+      "ManufacturersModelName": (
+         "RECOMMENDED",
+         "Manufacturer's designation of the eye-tracker model (for example"
+         '`"Eye-link 1000"`).'
+         ),
+      "SoftwareVersion": "RECOMMENDED",
+      "TaskDescription": "RECOMMENDED",
+      "Instructions": (
+         "RECOMMENDED", 
+         "Text of the instructions given to participants before the experiment."),
+      "CogAtlasID": "RECOMMENDED",
+      "CogPOID": "RECOMMENDED",
+      "DeviceSerialNumber": "RECOMMENDED",
+   }
+) }}
+
+
+Specific ET fields MUST be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "SamplingFrequency_eyetrack": "REQUIRED",
+      "SampleCoordinateUnit": "REQUIRED",
+      "SampleCoordinateSystem": "REQUIRED",
+      "EnvironmentCoordinates": "REQUIRED",
+      "ScreenSize": "REQUIRED",
+      "ScreenResolution": "REQUIRED",
+      "ScreenDistance": (
+         "REQUIRED",
+         "Screen distance in cm (e.g 60 for a screen distance of 60 cm), "
+         'if no screen use "n/a". '
+         "For MRI, it corresponds to the distance between the head-coil mirror "
+         "to the projection screen for example."),
+   }
+) }}
+
+Specific ET fields SHOULD be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "IncludedEyeMovementEvents": "RECOMMENDED",
+      "DetectionAlgorithm": "RECOMMENDED",
+      "DetectionAlgorithmSettings": "RECOMMENDED",
+      "StartMessage": "RECOMMENDED",
+      "EndMessage": "RECOMMENDED",
+      "KeyPressMessage": "RECOMMENDED",
+      "CalibrationType": "RECOMMENDED",
+      "CalibrationUnit": "RECOMMENDED",
+      "CalibrationPosition": "RECOMMENDED",
+      "MaximalCalibrationError": "RECOMMENDED",
+      "AverageCalibrationError": "RECOMMENDED",
+      "CalibrationList": "RECOMMENDED",
+      "RecordedEye": "RECOMMENDED",
+      "EyeCameraSettings": "RECOMMENDED",
+      "FeatureDetectionSettings": "RECOMMENDED",
+      "GazeMappingSettings": "RECOMMENDED",
+      "RawDataFilters": "RECOMMENDED",
+      "ScreenRefreshRate": "RECOMMENDED",
+      "AOIDefinition": "RECOMMENDED",
```

Comment:
Does AOI refers to Area od Interest? We may need to define it somewhere.
Also, is AOI equivalent to ROI (Region of Interest)? I think ROI is the option used in other parts of the specs

Reply:
Here this is something the community wanted to have, it refers to the screen and not to the brain... I would suggest we keep AOI to have the distinction clear.

Reply:
Then we can make some clarification, like Screen Area of Interest? SAOI or ScreenAOI. I would try to avoid terms with multiple meanings that could led to missinterpretation. What do you think?

Reply:
ok.

Reply:
I opted for `ScreenAOIDefinition`.